pkg/bitcoin/block.go

```diff
@@ -69,3 +105,70 @@ func (bh *BlockHeader) Hash() Hash {
 	//       3. Construct the Hash instance appropriately.
 	return Hash{}
 }
+
+// Target calculates the difficulty target of a block header.
+func (bh *BlockHeader) Target() *big.Int {
+	// A serialized 80-byte block header stores the `bits` value as a 4-byte
+	// little-endian hexadecimal value in a slot including bytes 73, 74, 75, and
```

Comment:
Why *hexadecimal*? If something is stored as bytes, it's just bytes.

Reply:
Since this function was updated and uses the `btcsuite` library now I removed this detailed explanation of how the 
target is calculated. I only added a simple and short description for this function.